### DISCUSSION TITLE
Moving towards 1.0.0

### DISCUSSION THREAD

**Discussion by sagikazarmark** [bernardphp/bernard#417] [Announcements]
Hey Bernard users!

Due to the recent activity in the project I decided to allocate some time to moving the library past the 1.0.0 milestone. It's long overdue and given the amount of effort and time many people (including me) invested in it it'd be a shame to see it perish.

Expect some changes in the coming days and weeks, including some potential breaking changes on the master branch.

Although I've been maintaining this library for a couple years now, I haven't really used it in the past couple years and I don't see that changing any time soon. As a result, we'll need community members to do extensive testing before tagging the final release. If you use this library in your (personal or company) projects, please consider applying as a beta tester.

Watch this space for news and changes!

Thanks,
Mark

**Comment by pluc** [comment]
Great news! I think the first important step would be to tag a 1.0 or 2.0 release depending on how you want to call it based off master so as not to break people's implementations. I'm currently implementing Bernard so if you tag a stable build I can work off it and let you know of any weirdness I encounter along the way

**Reply by pluc** [reply]
Ok so right now the master branch is broken - that's why I was suggesting tagging a 1.0 release before starting to mess with things.

```
$ composer require bernard/bernard:dev-master -W
    - symfony/event-dispatcher[v4.0.0, ..., v4.4.8] require php ^7.1.3 -> your php version (8.1.5) does not satisfy that requirement.
```

@sagikazarmark

Edit: Well, nevermind. Wasted two weeks implementing this just before you decided to mess it up, so I'll go find a stable library with an active maintainer instead of this.

**Reply by holtkamp** [reply]
@pluc https://github.com/symfony/event-dispatcher/blob/16178c629505a501f7367d7c73fb73a00f3b001b/composer.json#L19 now allows >=7.1.3

You got a point in tagging, but "dev-master" is allowed to be unstable. 

@sagikazarmark maybe create a new branche for the developments you intend to take place?

Also use this library for a while now, always works nicely 👍 , got little time, but maybe I can contribute

**Comment by holtkamp** [comment]
@sagikazarmark how to apply as a beta-tester?
